<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <attribute name="whatsThis">
        <string>&lt;p&gt;The Settings tab provides all bells and whistles required to fix support for modern day systems.&lt;/p&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0,1,0">
        <item row="4" column="0" colspan="3">
         <widget class="QGroupBox" name="settingsBox">
          <property name="title">
           <string>Game Tweaks</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="pathLabel">
          <property name="text">
           <string>Path:</string>
          </property>
          <property name="buddy">
           <cstring>gamePath</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="gamePath">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the installation path of the game here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="browseFilesButton">
          <property name="text">
           <string>&amp;Browse...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="gameSelect">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a game to edit the settings for here.&lt;/p&gt;&lt;p&gt;Support for more games can be added by installing the appropriate plugins.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="gameLabel">
          <property name="text">
           <string>&amp;Game:</string>
          </property>
          <property name="buddy">
           <cstring>gameSelect</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLabel" name="status">
          <property name="text">
           <string>&lt;font style=&quot;color: red;&quot;&gt;Directory not found.&lt;/font&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="saveAll">
          <property name="text">
           <string>Save Files...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="deviceTab">
       <attribute name="title">
        <string>Device Info</string>
       </attribute>
       <attribute name="whatsThis">
        <string>&lt;p&gt;This tab shows information about your graphics card(s).&lt;/p&gt;&lt;p&gt;Advanced users only.&lt;/p&gt;</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QStackedWidget" name="deviceStack">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="deviceInfo">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="deviceNameLabel">
              <property name="text">
               <string>Device:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="buddy">
               <cstring>deviceSelect</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="deviceSelect"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="deviceVendorLabel">
              <property name="text">
               <string>Vendor Id:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="deviceVendor">
              <property name="text">
               <string>0x0000</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="deviceIdLabel">
              <property name="text">
               <string>Device Id:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="deviceId">
              <property name="text">
               <string>0x0000</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="displayLabel">
              <property name="text">
               <string>Display:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="display">
              <property name="text">
               <string>Unknown</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="driverLabel">
              <property name="text">
               <string>Driver:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="driver">
              <property name="text">
               <string>unknown.dll</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="memoryLabel">
              <property name="text">
               <string>Texture memory:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="memory">
              <property name="text">
               <string>-1 MB</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="cardInDbLabel">
              <property name="text">
               <string>In Database:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="cardInDb">
              <property name="text">
               <string>&lt;font style=&quot;color: red;&quot;&gt;No!&lt;/font&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="noDevicesWarning">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>No DirectX graphics cards were detected! </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graphicsRulesTab">
       <attribute name="title">
        <string>GraphicRules.sgr</string>
       </attribute>
       <attribute name="whatsThis">
        <string>&lt;p&gt;This tab shows a preview of the generated graphic rules.&lt;/p&gt;&lt;p&gt;Advanced users only.&lt;/p&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="graphicsRulesSave">
          <property name="text">
           <string>&amp;Save As...</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="graphicsRuleSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="graphicsRulesText">
          <property name="font">
           <font>
            <family>Courier</family>
           </font>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="videoCardsTab">
       <attribute name="title">
        <string>VideoCards.sgr</string>
       </attribute>
       <attribute name="whatsThis">
        <string>&lt;p&gt;This tab shows the video card database and a preview of the file.&lt;/p&gt;&lt;p&gt;Advanced users only.&lt;/p&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <spacer name="videoCardsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>674</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="videoCardsSave">
          <property name="text">
           <string>&amp;Save As...</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSplitter" name="videoCardsSplitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="QTreeView" name="videoCardsView">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
             </widget>
             <widget class="QPlainTextEdit" name="videoCardsText">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Courier</family>
               </font>
              </property>
              <property name="undoRedoEnabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="copyright">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copyright © 2014-GRAPHICSRULESMAKER_COPYRIGHT_YEAR &lt;a href=&quot;http://www.simsnetwork.com&quot;&gt;SimsNetwork&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="locateGameAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="languageMenu">
     <property name="title">
      <string>&amp;Language</string>
     </property>
    </widget>
    <addaction name="languageMenu"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_English_Default">
   <property name="text">
    <string>&amp;English (Default)</string>
   </property>
  </action>
  <action name="action_Dutch_Nederlands">
   <property name="text">
    <string>&amp;Dutch (Nederlands)</string>
   </property>
  </action>
  <action name="locateGameAction">
   <property name="text">
    <string>&amp;Locate Game</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>mainTabs</tabstop>
  <tabstop>gameSelect</tabstop>
  <tabstop>gamePath</tabstop>
  <tabstop>browseFilesButton</tabstop>
  <tabstop>saveAll</tabstop>
  <tabstop>deviceSelect</tabstop>
  <tabstop>graphicsRulesText</tabstop>
  <tabstop>graphicsRulesSave</tabstop>
  <tabstop>videoCardsView</tabstop>
  <tabstop>videoCardsText</tabstop>
  <tabstop>videoCardsSave</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
